testbed/addons/karpenter/construct.ts

```diff
@@ -4,24 +4,25 @@ import iam = require('@aws-cdk/aws-iam');
 
 export interface KarpenterProps {
     cluster: eks.Cluster;
-    namespace?: string;
 }
 
 export class Karpenter extends cdk.Construct {
+    private namespace: string;
```

Comment:
Depending on your impl (props vs local), consider using a const instead of a private variable, since this value isn't used within member functions of this class. It's only used in the ctor.